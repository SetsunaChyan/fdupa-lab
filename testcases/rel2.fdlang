function main(){
i = input();
if(i > 100) {
    i = 100;
} else {
    nop;
}

j = i + 20;

while(i > 0) {
    i = i - 1;
    j = j - 1;
}

check_interval(j, 20, 20);
}