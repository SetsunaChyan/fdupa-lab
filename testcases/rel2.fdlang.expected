Line 16: YES; j in [20, 20]
